templates/cf-apache.service.mustache

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=CFEngine Enterprise Webserver
+After=syslog.target
+After=cf-postgres.service
+ConditionPathExists={{{vars.sys.workdir}}}/httpd/bin/apachectl
+PartOf=cfengine3.service
+
+[Service]
+Type=forking
+ExecStart={{{vars.sys.workdir}}}/httpd/bin/apachectl start
+ExecStop={{{vars.sys.workdir}}}/httpd/bin/apachectl stop
+PIDFile={{{vars.sys.workdir}}}/httpd/logs/httpd.pid
+Restart=always
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
+WantedBy=cfengine3.service
```

Comment:
Why keep an additional set of files here in masterfiles? This just makes it more complicated, and messes with files that the package manager is ultimately responsible for. I'm sure at some point someone will make a fix to those files in core and forget it here.